cadvisor.go

```diff
@@ -65,6 +65,8 @@ var whitelistedContainerLabels = flag.String("whitelisted_container_labels", "",
 
 var urlBasePrefix = flag.String("url_base_prefix", "", "prefix path that will be prepended to all paths to support some reverse proxies")
 
+var dockerOnlyPrefixWhiteList = flag.String("docker_only_prefix_whitelist", "", "A comma-separated list of cgroup path prefix that needs to be collected even when docker-only is specified")
```

Comment:
i'm not a particular fan of the naming for the `docker_only` flag, since it is runtime-specific.  Can we call it `raw_cgroup_roots` or something?  We can make it clear in the comment that setting it causes those cgroups to be collected even when --docker-only is specified since comments don't have backwards compatibility guarantees

Reply:
@dashpole would raw_cgroup_prefix_whitelist be a better fit?

Reply:
that would be fine as well